src/test/run-pass/rfcs/rfc-2126-extern-absolute-paths/basic.rs

```diff
@@ -7,7 +7,6 @@
 use xcrate::Z;
 
 fn f() {
-    use xcrate;
```

Comment:
The import here is a significant part of the test, so the warning needs to be `allow`ed.